src/main/java/org/cloudfoundry/promregator/scanner/ReactiveTargetResolver.java

```diff
@@ -352,7 +356,7 @@ public void setResolvedApplicationId(String resolvedApplicationId) {
 					.filter(appResource -> appNameToSearchFor.equals(appResource.getEntity().getName().toLowerCase(Locale.ENGLISH)))
 					.single()
 					.doOnError(e -> {
-						if (e instanceof NoSuchElementException) {
+						if (e instanceof NoSuchElementException && this.emptyResolutionIsLogged) {
 							log.warn(String.format("Application id could not be found for org '%s', space '%s' and application '%s'. Check your configuration; skipping it for now", it.getResolvedOrgName(), it.getResolvedSpaceName(), it.getConfigTarget().getApplicationName()));
```

Comment:
y not just create a sub-logger that the user could disable via normal logging config?

Reply:
To be honest, I was unsure about that approach myself. 
I had thought about this, but the configuration open then would look very strange:

```
logging.org.cloudfoundry.promregator.scanner.ReactiveTargetResolver.EmptyTargets=ERROR
```

It is simply very hard to explain in documentation.

The other would have been to keep the configuration approach as it is right now, but then to have a sublogger, whose level then needs to be tweaked during initialization. However that could also be wrong, because of someone then writes 

```
logging.org.cloudfoundry.promregator.scanner.ReactiveTargetResolver.EmptyTargets=DEBUG
```

into his config, then we also have conflicting statements there. 

So, this all wasn't really convincing to.
Or what do you think?

Reply:
I would have it log at `info` level and if the user wishes to mute it they will most likely fix the config to not scan for that app by name (our ideal solution), but if they actually want to keep scanning for the missing target, they will noise in the logs and look for the typical way they would mute logging (controlling the logger level), by setting it to WARN or ERROR.

I suspect they wouldn't even think to look for the `promregator.resolver.logging.empty.target` property.

So my recommendation would be to not include `promregator.resolver.logging.empty.target`, but even if it is included, also separate out the noisy logger so it can be muted via typical means, without muting the entire class.

Reply:
True that: People will be coming from the logger's name, not from the configuration page. You convinced me - I'll change it.